src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta

```diff
@@ -64,6 +64,9 @@
                 this.hiddenInputsName = config.hiddenInputsName;
 
                 function refreshHiddenInputs() {
+                    if (input.hiddenInputsName === undefined) {
```

Comment:
```suggestion
                    if (typeof input.hiddenInputsName === "undefined") {
```

Reply:
would something like `if (!input.hiddenInputsName)` not be the most simple solution?

or what is the advantage of this "typeof"-check?